### PR TITLE
Feature/improve image

### PR DESCRIPTION
**Pull request by nowak-pawel-ds**

### CODE REVIEW

File: ui.apps/src/main/jcr_root/apps/bulma/components/card/_ws_dialog/.content.xml [hunk 1]
```diff
@@ -30,63 +30,59 @@
...
-                    sling:resourceType="wcm/dialogs/components/textfield"
-                    name="urls/1/label"
-                    label="Label"/>
-            <url2
+            <urls
+            jcr:primaryType="nt:unstructured"
+            name="urls"
+            label="Footer URLs"
+            sling:resourceType="wcm/dialogs/components/multifield">
+                <adress
```
Comment (marcell-pal-ds) [inline ui.apps/src/main/jcr_root/apps/bulma/components/card/_ws_dialog/.content.xml]: Typo in 'adress'

File: ui.apps/src/main/jcr_root/apps/bulma/components/card/common/image/.content.xml [hunk 2]
```diff
@@ -7,19 +7,57 @@
...
             label="Alt text"/>
+    <style
+        jcr:primaryType="nt:unstructured"
+        sling:resourceType="wcm/dialogs/components/include">
+        <include
+            jcr:primaryType="nt:unstructured"
+            sling:resourceSuperType="/apps/bulma/components/image/attributes/ratio"
+            name="image/style"
+            label="Image ratio">
+            <is-4by3
```
Comment (marcell-pal-ds) [inline ui.apps/src/main/jcr_root/apps/bulma/components/card/common/image/.content.xml:33]: You could also use the `sling:hideChildren` attribute this way:
`sling:hideChildren="[is-4by3,is-3by5,...]"`

This is maybe a more readable format, but of course feel free to use the already implemented way as the result should be the same anyways.

**Review by marcell-pal-ds — CHANGES_REQUESTED** [review]

File: ui.apps/src/main/jcr_root/apps/bulma/components/card/common/image/.content.xml (hunk 2 above)
Comment (nowak-pawel-ds) [reply ui.apps/src/main/jcr_root/apps/bulma/components/card/common/image/.content.xml:33]: I tried it on include tag and it did not work :/ where should I insert it?

Comment (marcell-pal-ds) [reply ui.apps/src/main/jcr_root/apps/bulma/components/card/common/image/.content.xml:33]: This code snippet works for me:
image

I also realized, that putting whitespace between multiple elements does not work, so if you have more elements, you need to use simply the commas like I did in the example of my first comment here.

Comment (nowak-pawel-ds) [reply ui.apps/src/main/jcr_root/apps/bulma/components/card/common/image/.content.xml:33]: it did not work for me :/ tried it

Comment (marcell-pal-ds) [reply ui.apps/src/main/jcr_root/apps/bulma/components/card/common/image/.content.xml:33]: Alright.

**Review by marcell-pal-ds — APPROVED** [review]

**Review by arturwujczak — APPROVED** [review]

**Review by paweljozwicki — APPROVED** [review]

**Pull request merged**